website/content/docs/secrets/gcp.mdx

```diff
@@ -211,16 +211,37 @@ below.
 For more information on creating and managing static accounts, see the
 [GCP secrets engine API docs][api] docs.
 
+## Impersonated Accounts
```

Comment:
Maybe we can link to GCP docs on impersonation somewhere in this paragraph? Perhaps to https://cloud.google.com/iam/docs/impersonating-service-accounts.

Reply:
Sounds like a good idea to me. Added here: https://github.com/hashicorp/vault/pull/18519/files#diff-d9726e27b0de244125c80192a0fef3936511f40caf6aa37885d1976a9bc9a92fR223